clang/include/clang/Driver/Driver.h

```diff
@@ -602,6 +602,16 @@ class Driver {
   // FIXME: This should be in CompilationInfo.
   std::string GetProgramPath(StringRef Name, const ToolChain &TC) const;
 
+  /// GetModuleManifestPath - Lookup the name of the Standard library manifest.
+  ///
+  /// \param C - The compilation.
+  /// \param TC - The tool chain for additional information on
+  /// directories to search.
+  //
+  // FIXME: This should be in CompilationInfo.
```

Comment:
If possible, let's explain the reason why we choose to put it here.

Reply:
I copied it from the functions above, I have not investigated why the FIXME is there.

Reply:
Got it. I didn't look into the context.

Reply:
Remove the FIXME. `CompilationInfo` has been removed long ago.